classes/PodsForm.php

```diff
@@ -229,6 +229,99 @@ public static function field( $name, $value, $type = 'text', $options = null, $p
     }
 
     /**
+     * Output a flexible relationship button
+     *
+     * @param array $field Field options array
+     *
+     * @param int   $item_id
+     *
+     * @return string Button HTML
+     *
+     * @since 2.7
+     */
+    public static function flex_relationship( $field, $item_id ) {
+
+        $output = '';
+
+        // Early exit if any of:
+        //  * Not admin
+        //  * this isn't a flexible relationship enabled field
+        //  * we're already in a modal
+        if ( ! is_admin() || empty( $field[ 'type' ] ) || 'pick' != $field[ 'type' ] || empty( $field[ 'pick_flexible' ] ) || pods_is_modal_window() ) {
+            return $output;
+        }
+
+        wp_enqueue_style( 'responsive-modal' );
+        wp_enqueue_script( 'responsive-modal' );
+
+        // Set the file name and args based on the content type of the relationship
+        switch ( $field[ 'pick_object' ] ) {
```

Comment:
We need to add permissions checks for each these too, maybe a separate switch statement?